routing/car_directions.cpp

```diff
@@ -30,24 +30,25 @@ void FixupCarTurns(vector<RouteSegment> & routeSegments)
   // (1) the route enters to the roundabout;
   // (2) the route leaves the roundabout;
   uint32_t exitNum = 0;
-  size_t currentEnterRoundAbout = routeSegments.size();
+  size_t const kInvalidEnter = routeSegments.size();
```

Comment:
We use 'k' prefix for the compile time constants, but here 'invalidEnter' is better.

Reply:
OK